elcid/utils.py

```diff
@@ -81,11 +82,30 @@ def natural_keys(text):
     return [ atoi(c) for c in re.split(r'(\d+)', text) ]
 
 
+def get_or_create_patient(mrn, episode_category, run_async=None):
+    from intrahospital_api import loader
```

Comment:
Honestly I'd be tempted to put this in the intrahospital_api if we're doing this kind of thing